framework/framework.go

```diff
@@ -454,3 +499,21 @@ func ProcessUpdate(ctx context.Context, obj interface{}, resources []Resource) e
 
 	return nil
 }
+
+func (f *Framework) bootWithError() error {
+	if f.tpr != nil {
+		err := f.tpr.CreateAndWait()
+		if tpr.IsAlreadyExists(err) {
+			f.logger.Log("debug", "third party resource already exists")
```

Comment:
There is a log message that third party resource already exists but no logs about creating it or when it is actually created.

Reply:
Added more logs.